python/tensorboard/summary.py

```diff
@@ -183,7 +183,7 @@ def image(tag, tensor):
 def make_image(tensor, height, width, channel):
     """Convert an numpy representation image to Image protobuf"""
     image = Image.fromarray(tensor)
-    output = StringIO.StringIO()
+    output = StringIO() # This is working for python 2.7
```

Comment:
I am using Anaconda python 2.7. I got errors with this error `AttributeError: class StringIO has no attribute 'StringIO'
`
When I made the modification, it is working.